### ISSUE TITLE
FIXME: NSSUTIL issues on Linux

### ISSUE THREAD

**Issue by joseasoler** [oceancabbage/RimSort#59]
On a recently updated ArchLinux, trying to run the RimSort-Alpha-v1.0.3-RC3 binary in releases triggers this error:

```
❯ ./RimSort.bin
RimSort.py: <_MainProcess name='MainProcess' parent=None started>
__name__: __main__
sys.argv: ['./RimSort.bin']
steamworks.wrapper: <_MainProcess name='MainProcess' parent=None started>
__name__: util.steam.steamworks.wrapper
sys.argv: ['./RimSort.bin']
Traceback (most recent call last):
  File "/tmp/onefile_17318_1680856464_975258/RimSort.py", line 20, in <module>
  File "/tmp/onefile_17318_1680856464_975258/view/main_content_panel.py", line 34, in <module view.main_content_panel>
  File "/tmp/onefile_17318_1680856464_975258/window/web_content_panel.py", line 7, in <module window.web_content_panel>
ImportError: /tmp/onefile_17318_1680856464_975258/PySide6/../libnssutil3.so: version `NSSUTIL_3.82' not found (required by /usr/lib/libnss3.so)
```

My system has this version of the nss library:

```
❯ paru -Qs | grep 'local/nss'
local/nss 3.89-1
```

**Comment by twstagg** [comment]
Issue is resolved if you remove `libnssutil3.so` entirely from the build. However, I do not think this is possible using --onefile at this time. I am currently researching options to exclude this at at the time of `nuitka` build

**Comment by twstagg** [comment]
* [This issue](https://github.com/Nuitka/Nuitka/issues/2154) was fixed in Nuitka 1.6
            * At this time (04/09/2023) you can install with nuitka factory using `pip install -U --force-reinstall "https://github.com/Nuitka/Nuitka/archive/factory.zip"`

Builds from a1.0.3-RC5 and later will utilize this version of nuitka, which contain the ultimate fix upstream for this issue.
